hack/aks/property-based-scheduling.md

```diff
@@ -0,0 +1,428 @@
+# Use property-based scheduling with Fleet on AKS clusters: Preview
```

Comment:
Insert this header at the top of article:

```
ms.topic: article
ms.custom: azure-kubernetes-service, devx-track-azurecli
ms.date: 04/19/2024
author: michaelawyu
```

Reply:
Done.